tools/ci_build/github/apple/assemble_ios_packaging_artifacts.sh

```diff
@@ -49,7 +49,11 @@ assemble_and_upload_pod() {
   popd
 }
 
-assemble_and_upload_pod "onnxruntime-mobile-c"
-assemble_and_upload_pod "onnxruntime-mobile-objc"
-assemble_and_upload_pod "onnxruntime-c"
-assemble_and_upload_pod "onnxruntime-objc"
+# upload the pod archives and podspecs for each pod in parallel
```

Comment:
does an error in one of these background jobs still cause the whole script to fail? we want to keep that property.
if it is simpler to just run these sequentially, that's fine. these steps aren't expected to take very long.

Reply:
If one of the background jobs fails, the other jobs should continue running. The wait command theoretically waits for all jobs to finish/terminate before exiting.

Reply:
https://www.gnu.org/software/bash/manual/bash.html#index-wait

"If no arguments are given, wait waits for all running background jobs and the last-executed process substitution, if its process id is the same as $!, and the return status is zero."

this seems to indicate that the exit code will be successful. we want the script to exit with a non-zero exit code if one of these fails so the CI build will fail.